Signal/src/util/OWSScreenLock.swift

```diff
@@ -323,7 +315,10 @@ import LocalAuthentication
     private func screenLockContext() -> LAContext {
         let context = LAContext()
 
-        context.touchIDAuthenticationAllowableReuseDuration = TimeInterval(screenLockTimeout())
+        // If user has set any non-zero timeout, recycle biometric auth
+        // in the same period as our normal screen lock timeout.
```

Comment:
Comment is misleading since it's min(10, x)

Reply:
Done.